entrypoint.sh

```diff
@@ -1,3 +1,24 @@
 #!/bin/sh
 
+function send_test_failure_notification {
+    if [[ -z "${SLACK_URL}" ]]; then
+        echo "Emulating sendTestFailureNotification"
+        
+    else
+        curl -X POST -H 'Content-type: application/json' \
```

Comment:
Add link to logs

Reply:
https://console.cloud.google.com/logs/query;query=resource.type%3D%22k8s_container%22%0Aresource.labels.project_id%3D%22entur-1287%22%0Alabels.k8s-pod%2Fapp%3D%22geocoder-acceptance-tests%22;timeRange=PT1H;cursorTimestamp=2021-05-15T20:15:18.677405046Z?project=entur-1287